riptide_rviz/riptide_rviz/python/MarkerPublisher.py

```
@@ -284,10 +299,57 @@ def timerCB(self):
+        
+        ghostBaseLinkPose = Pose()
+        ghostBaseLinkPose.position.x = ghost.pose.position.x
+        ghostBaseLinkPose.position.y = ghost.pose.position.y
+        ghostBaseLinkPose.position.z = ghost.pose.position.z
+        ghostBaseLinkPose.orientation.w = ghost.pose.orientation.w
+        ghostBaseLinkPose.orientation.x = ghost.pose.orientation.x
+        ghostBaseLinkPose.orientation.y = ghost.pose.orientation.y
+        ghostBaseLinkPose.orientation.z = ghost.pose.orientation.z
+        
+        # Rotate the base_link offset vector from the robot frame 
+        # into the world frame using the robot’s orientation.
```

Ruff (0.15.20)

[warning] Comment contains ambiguous `’` (RIGHT SINGLE QUOTATION MARK). Did you mean ``` (GRAVE ACCENT)?

(RUF003)